llvm/test/CodeGen/AMDGPU/andorn2.ll

```diff
@@ -25,6 +25,28 @@ entry:
   ret void
 }
 
+; GCN-LABEL: {{^}}scalar_andn2_i32_one_sgpr
+; GCN: s_andn2_b32
+define amdgpu_kernel void @scalar_andn2_i32_one_sgpr(
```

Comment:
better to use a shader calling convention and use the return value. the style of tests with the output pointer and a kernel is older and predates function support. kernels have a lot of extra noise in the output

Reply:
Okay, I updated it. :)